src/IonParserTextRaw.ts

```diff
@@ -1065,54 +1111,27 @@ private _test_symbol_as_annotation() : boolean {
           acceptComments = false;
           for(index = this._start; index < this._end; index++) {
               ch = this._in.valueAt(index);
-              switch (ch) {
-                  case CH_BS:
-                      escaped = this.readClobEscapes(index, this._end);
-                      if(escaped >= 0){
-                          s += String.fromCharCode(escaped);
-                      }
-                      index += this._esc_len;
-                      break;
-                  case CH_SQ:
-                      if(this.verifyTriple(index)){
-                          index = this._skip_triple_quote_gap(index, this._end, acceptComments);
-                      } else {
-                          s += String.fromCharCode(ch);
-                      }
-                      break;
-
-                  default:
+              if(ch === CH_BS) {
+                  escaped = this.readClobEscapes(index, this._end);
+                  if (escaped >= 0) {
+                      s += String.fromCharCode(escaped);
+                  }
+                  index += this._esc_len;
+                  break;
+              }else if ( ch ===CH_SQ) {
```

Comment:
Missing space between } and e

There are a few other instances of this in this diff too.